### ISSUE TITLE
Traffic sign enhancements

### ISSUE THREAD

**Issue by thuryn** [osmandapp/OsmAnd#10401]
I would like to see more traffic signs when I'm approaching them like the STOP sign - at least Give way sign, City limit sign (with name attribute insight). Maybe more.
Next - when there are two signs close together, the second sign is not shown or it is but for a second. I mean when for example speed limit follows stop sign after 30m, the stop sign is show just for a second. It would be great to show both signs together or one after the other but for at least 5 seconds or more.

**Comment by vshcherb** [comment]
That's interesting, though as of today we can do it only for navigation when the route is calculated and we have full information which direction / lane vehicle is moving to

**Comment by thuryn** [comment]
Yes, that's the case. I haven't written it so precisely, bout you are right.

**Comment by thuryn** [comment]
One more feature related to traffic signs - it would be greater to show (as an option) traffic signs on map also like Vespucci app does.
![Screenshot_2020-12-20-12-21-32-530_de blau android](https://user-images.githubusercontent.com/11028346/102712003-29f3e000-42be-11eb-8c2f-975780c37925.jpg)

![Screenshot_2020-12-20-12-21-32-530_de blau android](https://user-images.githubusercontent.com/11028346/102712009-34ae7500-42be-11eb-9f6e-bf0fe12bcedd.jpg)

**Comment by thuryn** [comment]
And the crossing
![Screenshot_2020-12-20-17-02-08-976_de blau android](https://user-images.githubusercontent.com/11028346/102717969-caa9c600-42e5-11eb-8699-7e536452d625.jpg)
And City limit (with name tag):
![Screenshot_2020-12-20-17-01-12-227_de blau android](https://user-images.githubusercontent.com/11028346/102717989-e9a85800-42e5-11eb-83f0-cbd0aadfc307.jpg)

**Comment by anastasiia936** [comment]
Add give-of-way sign alert.
